benchmarks/2022.json

```diff
@@ -22,13 +23,12 @@
     "measureId": "001",
     "benchmarkYear": 2020,
     "performanceYear": 2022,
-    "submissionMethod": "claims",
+    "submissionMethod": "cmsWebInterface",
     "isToppedOut": false,
-    "isHighPriority": true,
     "isToppedOutByProgram": false,
     "deciles": [
       100,
-      80,
```

Comment:
This is just a reordering of 001's claims and WI benchmarks. Not sure why, but no action needed.